target/linux/ramips/image/mt7621.mk

```diff
@@ -431,15 +431,26 @@ define Device/bolt_arion
 endef
 TARGET_DEVICES += bolt_arion
 
-define Device/cudy_wr1300
+define Device/cudy_wr1300-v1
   $(Device/dsa-migration)
   IMAGE_SIZE := 15872k
   DEVICE_VENDOR := Cudy
   DEVICE_MODEL := WR1300
+  DEVICE_VARIANT := V1
   DEVICE_PACKAGES := kmod-mt7603 kmod-mt76x2 kmod-usb2 kmod-usb3 \
 	kmod-usb-ledtrig-usbport
```

Comment:
I forgot about it, add here `SUPPORTED_DEVICES += cudy,wr1300`, since current images for v1 use old compatible and without it sysupgrade will trigger a warning about incompatible image.

BTW You could probably experiment by appending to `SUPPORTED_DEVICES` R10 for v1 and R23 for v2 which are supported strings in vendor firmware. Then vendor firmware probably won't complain about incompatible image.

Reply:
Done, but as I say in my comment below the string `cudy,wr1300` will have to be in both versions if we want to avoid the message when an OpenWRT user upgrades from the current version to the version that will be generated after applying this change.